src/test/scala/com/microsoft/hyperspace/index/rules/RuleUtilsTest.scala

```diff
@@ -143,6 +144,10 @@ class RuleUtilsTest extends HyperspaceRuleTestSuite with SQLHelper {
             if (expectCandidateIndex) {
               assert(indexes.length == 1)
               assert(indexes.head.name == "index1")
+              assert(
+                indexes.head
+                  .getTagValue(plan, IndexConstants.INDEX_HYBRIDSCAN_REQUIRED_TAG)
+                  .getOrElse(false) == expectedHybridScanTag)
```

Comment:
Can you test the return value of `getTagValue`? You can make `expectedHybridScanTag` as `Option[Boolean]` if needed.